cli/cmd/compliance_aws.go

```diff
@@ -448,7 +448,7 @@ The output from status with the --json flag can be used in the body of PATCH api
 					Csp:     api.AwsInventoryType,
 				}
 			)
-			err := cli.LwApi.V2.Inventory.Search(&awsInventorySearchResponse, filter)
+			err := api.WindowedSearch(cli.LwApi.V2.Inventory.Search, 7, 92, &awsInventorySearchResponse, &filter)
```

Comment:
Can we set constants with the real min, max values rather than arbitrary numbers?

Reply:
Yes, I'll create consts